src/platform_impl/linux/window.rs

```diff
@@ -102,11 +108,13 @@ impl Window {
     window.set_default_size(1, 1);
     window.resize(width, height);
 
-    if attributes.maximized {
-      window.maximize();
-    }
+    let maximize_process =
+      util::WindowMaximizeProcess::new(window.clone(), attributes.maximized, attributes.resizable);
+    glib::timeout_add_local(std::time::Duration::from_millis(25), move || {
```

Comment:
instead of using a `timeout_add_local`, maybe it is better to use `idle_add_full` with a priority of `PRIORITY_HIGH_IDLE + 10` so it runs on the next iteration instead of a timeout

see https://docs.gtk.org/glib/func.idle_add_full.html and https://docs.gtk.org/glib/const.PRIORITY_HIGH_IDLE.html

Reply:
You’re right! It seems that the priority assigned when registering to the main loop is more important than using a timeout. When creating a new window, `Priority::HIGH_IDLE` works, but it causes a "Protocol Error 71" in the event loop. So, for the maximize event in the event loop, I applied `Priority::DEFAULT_IDLE`.